tools/test-proxy/Azure.Sdk.Tools.TestProxy/Playback.cs

```diff
@@ -25,9 +25,17 @@ public sealed class Playback : ControllerBase
         [HttpPost]
         public async Task Start()
         {
-            string file = RecordingHandler.GetHeader(Request, "x-recording-file");
-
-            await _recordingHandler.StartPlayback(file, Response);
+            string file = RecordingHandler.GetHeader(Request, "x-recording-file", true);
+            string recordingId = RecordingHandler.GetHeader(Request, "x-recording-id", true);
+
+            if (String.IsNullOrWhiteSpace(file) && !String.IsNullOrWhiteSpace(recordingId))
```

Comment:
```suggestion
            if (String.IsNullOrEmpty(file) && !String.IsNullOrEmpty(recordingId))
```

NIT: In .NET we typically just check `String.IsNullOrEmpty()`.  If a caller passes a string of whitespace, it's their problem.  There are a lot of possible invalid strings, and all-whitespace is just one example we usually don't special-case.

Reply:
Gotcha. Will resolve everwhere I use this.